stan/math/prim/prob/cauchy_lccdf.hpp

```diff
@@ -32,31 +32,27 @@ template <typename T_y, typename T_loc, typename T_scale>
 return_type_t<T_y, T_loc, T_scale> cauchy_lccdf(const T_y& y, const T_loc& mu,
                                                 const T_scale& sigma) {
   using T_partials_return = partials_return_t<T_y, T_loc, T_scale>;
-
-  if (size_zero(y, mu, sigma)) {
-    return 0.0;
-  }
-
   static const char* function = "cauchy_lccdf";
-
-  T_partials_return ccdf_log(0.0);
-
   check_not_nan(function, "Random variable", y);
   check_finite(function, "Location parameter", mu);
   check_positive_finite(function, "Scale parameter", sigma);
   check_consistent_sizes(function, "Random variable", y, "Location parameter",
                          mu, "Scale Parameter", sigma);
 
+  if (size_zero(y, mu, sigma)) {
+    return 0;
+  }
+
+  using std::atan;
```

Comment:
[comment]
Just a comment because you're fixing so much of the style, but not worth going in and changing for this PR.  

I'd like to see using statements either all piled together at the top of the function or pushed down to just before the symbol is used.

Reply:
Up to now they were placed inconsistently, at least my change put them in the same place.  I can move them to the top in the future.